### PR TITLE
Disable navigating back with Backspace/Delete key

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. Could you also confirm it works for Macs? 
Staged at https://apps-stage-2.risevision.com. 
Thanks!

### CODE REVIEW

File: src/js/dtv-common-header.js [hunk 1]
```diff
@@ -123,25 +123,33 @@ angular.module("risevision.common.header", [
...
+  "analyticsEvents", "$document",
   function (segmentAnalytics, SEGMENT_API_KEY, ENABLE_INTERCOM_MESSAGING,
-    analyticsEvents) {
+    analyticsEvents, $document) {
     analyticsEvents.initialize();
     segmentAnalytics.load(SEGMENT_API_KEY, ENABLE_INTERCOM_MESSAGING);
+
+    $document.on("keydown", function (e) {
+      var nodeName = e.target.nodeName.toLowerCase();
+      if (e.which === 8 && nodeName !== "input") {
```
Comment (olegrise) [inline src/js/dtv-common-header.js]: @ezequielc why not check for all possible field types. See example http://stackoverflow.com/questions/1495219/how-can-i-prevent-the-backspace-key-from-navigating-back?answertab=votes#tab-top

Comment (ezequielc) [inline src/js/dtv-common-header.js]: @olegrise Makes sense, I will change that. I will add the code for disabled and read only too. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc confirmed delete button is not causing the page to go back in history anymore on the Mac. Thanks.

**Comment by ezequielc** [comment]
Thanks @alex-deaconu !
@olegrise I implemented the suggested changes. 
Staged at https://apps-stage-2.risevision.com. 
Please review. Thanks

**Comment by olegrise** [comment]
👍

**Comment by alex-deaconu** [comment]
@ezequielc looks good. Verified with a few elements like checkboxes and disabled text boxes and it worked like a charm. Thanks for fixing 👍  !!

**Comment by ezequielc** [comment]
Thanks @olegrise @alex-deaconu and @RV-Adrian for the help! Cheers

**Pull request merged**